src/elli_http.erl

```diff
@@ -58,7 +60,7 @@ start_link(Server, ListenSocket, Options, Callback) ->
       Options      :: proplists:proplist(),
       Callback     :: elli_handler:callback().
 accept(Server, ListenSocket, Options, Callback) ->
-    case catch elli_tcp:accept(ListenSocket, Server, accept_timeout(Options)) of
+    case elli_tcp:accept(ListenSocket, Server, accept_timeout(Options)) of
```

Comment:
I don't see where `catch` was actually used, since all the clauses derive not from exceptions, but simple function returns.